packages/frontend/src/components/PieChartConfig/PieChartSeriesConfig.tsx

```diff
@@ -286,39 +346,110 @@ const PieChartSeriesConfig: FC = () => {
-                    p="sm"
-                    sx={(theme) => ({ borderRadius: theme.radius.sm })}
-                >
-                    {groupLabels.map((groupLabel) => (
-                        <GroupItem
-                            key={groupLabel}
-                            swatches={defaultColors}
-                            defaultColor={groupColorDefaults[groupLabel]}
-                            defaultLabel={groupLabel}
-                            color={groupColorOverrides[groupLabel]}
-                            label={groupLabelOverrides[groupLabel]}
-                            valueLabel={
-                                groupValueOptionOverrides[groupLabel]
-                                    ?.valueLabel ?? valueLabel
-                            }
-                            showValue={
-                                groupValueOptionOverrides[groupLabel]
-                                    ?.showValue ?? showValue
-                            }
-                            showPercentage={
-                                groupValueOptionOverrides[groupLabel]
-                                    ?.showPercentage ?? showPercentage
-                            }
-                            onLabelChange={groupLabelChange}
-                            onColorChange={groupColorChange}
-                            onValueOptionsChange={groupValueOptionChange}
-                        />
-                    ))}
-                </Stack>
+            {sortedGroupLabels.length === 0 ? null : (
+                <DragDropContext onDragEnd={handleDragEnd}>
+                    <Droppable droppableId="droppable">
+                        {(droppableProvided, droppableSnapshot) => (
+                            <Box
+                                ref={droppableProvided.innerRef}
+                                bg={
+                                    droppableSnapshot.isDraggingOver
+                                        ? 'gray.1'
+                                        : 'gray.0'
+                                }
+                                p="sm"
+                                sx={(theme) => ({
+                                    borderRadius: theme.radius.sm,
+                                })}
+                            >
+                                {sortedGroupLabels.map((groupLabel, index) => (
+                                    <Draggable
+                                        key={groupLabel}
+                                        draggableId={groupLabel}
+                                        index={index}
+                                    >
+                                        {(
+                                            draggableProvided,
+                                            draggableSnapshot,
+                                        ) => (
+                                            <GroupItem
+                                                key={groupLabel}
```

Comment:
Do these still need keys no when wrapped in Draggable? I guess it doesn't hurt.

Reply:
totally useless 👍